### PR TITLE
Athena bug solved

### LINKED ISSUE #10089: [Bug]: Athena Handler Connection Errors Only Visible in Docker Logs, Not in MindsDB Interface
### Short description of current behavior

Description: When the Athena handler in MindsDB fails to connect (e.g., due to an invalid endpoint), the error message only appears in Docker logs (Failed to connect to Athena: Invalid endpoint). However, this error does not propagate to the MindsDB interface, leaving users unaware of the misconfiguration unless they check the logs.

### Video or screenshots

image
image

### Expected behavior

Expected Behavior: Connection errors should appear directly in the MindsDB interface, providing immediate feedback to users regarding connection issues.

Observed Behavior: Errors are logged in Docker only, requiring users to monitor logs to identify the source of the problem.

### How to reproduce the error

1. Create database with faulty or empty parameter
```bash
CREATE DATABASE athena_datasource
WITH
engine='athena',
parameters={
    'aws_access_key_id': '',
    'aws_secret_access_key': '',
    'region_name': '',
    'catalog': '',
    'database': '',
    'workgroup': '',
    'results_output_location': '',
    "check_interval": '0'
};
```
2. Check the MindsDB interface and docker logs.

### Anything else?

Suggested Solution: Update the Athena handler to display error messages in the MindsDB interface for easier troubleshooting.

### PR DESCRIPTION
**Pull request by parthiv11**
## Description
- simply updated `HandlerStatusResponse.sucess=False` of exception occured

Fixes #10089

## Type of change

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)

## Additional Media:

- [x] I have attached a brief loom video or screenshots showcasing the new functionality or change.
- [ ] 
image

## Checklist:

- x] My code follows the style guidelines(PEP 8) of MindsDB.

### CODE REVIEW

File: mindsdb/integrations/handlers/athena_handler/athena_handler.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def connect(self) -> StatusResponse:
             return StatusResponse(self.name, True, 'Connected successfully')
```
Comment (ZoranPandovski) [inline mindsdb/integrations/handlers/athena_handler/athena_handler.py:62]: Can we please update this too? Also there are 2 more places we need to update in this class

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]

File: mindsdb/integrations/handlers/athena_handler/athena_handler.py (hunk 1 above)
Comment (parthiv11) [reply mindsdb/integrations/handlers/athena_handler/athena_handler.py]: done !!

**Review by ZoranPandovski — CHANGES_REQUESTED** [review]
Pelase fix flake8 warnings

**Comment by parthiv11** [comment]
done!!

**Review by ZoranPandovski — APPROVED** [review]

**Pull request merged**